deploy/examples/create-external-cluster-resources-tests.py

```diff
@@ -240,11 +240,22 @@ def test_skip_monitoring_endpoint_no_prometheus(self):
         del cmd_json_out["mgrmap"]["services"]["prometheus"]
         self.rjObj.cluster.cmd_output_map[cmd_key] = json.dumps(cmd_json_out)
 
-        endpoint, port = self.rjObj.get_active_and_standby_mgrs()
+        endpoint = ""
+        port = ""
+        try:
+            endpoint, port = self.rjObj.get_active_and_standby_mgrs()
+            self.fail("An Exception was expected to be thrown")
```

Comment:
Was this failure just for testing and needs to be reverted?

Reply:
NO these are part of the changes,
As of now instead of returning, we will be raising an exception,

So we need to catch that in the unit test too

Reply:
Ok I see now the purpose, thanks